doc/adding_features.md

```diff
@@ -483,30 +502,95 @@ class SomeHandler(BaseHandler):
+
+
+After adding a new handler and/or model, we ensure that they can be imported in other modules by adding them to the `__init__.py` files in their respective directories. For example, for `skyportal/handlers/api/test_comment.py`, we add the following line to the `skyportal/handlers/api/__init__.py` file:
+
+``` python
+from .test_comment import TestCommentHandler
+```
+
+and for `skyportal/models/test_comment.py`, we add the following line to the `skyportal/models/__init__.py` file:
+
+``` python
+from .test_comment import TestComment
+```
+
+Now, we need to define the API endpoint that will be used to access the `TestCommentHandler` we just defined. This is done in the `skyportal/app_server.py` file.
+
+``` python
+from skyportal.handlers.api import TestCommentHandler # add this line to the top of the file after the existing handlers imports
+
+...
+
+skyportal_handlers = [
+    ...
+    (r'/api/test_comment', TestCommentHandler), # add this line in the skyporatl_handlers list
+    ...
+]
+```
+
+The elements of this `skyportal_handlers` list are tuples consisting of a regex string describing the matching URL pattern, and the handler object: `(r'/api/test_comments', TestCommentHandler)`. In this case, we want to access the `TestCommentHandler` at the `/api/test_comment` path.
 
-After adding the relevant import lines to `skyportal/handlers/api/__init__.py` and `skyportal/app_server.py` to import our new `TestCommentHandler` into the `app_server` module, we then add an element to our endpoint & handler mapping list (see the `handlers` assignment statement in the `make_app` function defined in `skyportal/app_server.py`) to map the associated API endpoint to the new handler class. The elements of this list are tuples consisting of a regex string describing the matching URL pattern, and the handler object: `(r'/api/test_comments, TestCommentHandler')`. After defining our handler and mapping it to the API endpoint in our application, we can now expect any authenticated GET or POST requests to `<base_URL>/api/test_comments` to either fetch or create records.
+Keep in mind that incomign requests will be routed to the first handler in the list that matches the request path. When you add a new feature, make sure to add it to the list in the correct place so that it doesn't override an existing handler.
```

Comment:
```suggestion
Keep in mind that incoming requests will be routed to the first handler in the list that matches the request path. When you add a new feature, make sure to add it to the list in the correct place so that it doesn't override an existing handler (unless that is what you intend to do!). 
```